crates/sui-graphql-rpc/src/types/open_move_type.rs

```diff
@@ -44,7 +44,7 @@ type OpenMoveTypeSignatureBody =
   | \"u8\" | \"u16\" | ... | \"u256\"
   | { vector: OpenMoveTypeSignatureBody }
   | {
-      struct {
+      datatype {
```

Comment:
Don't you also need to change the `Struct` on line 86?

Reply:
I do indeed...Will do

Reply:
Should be done now